src/api/graphql/events/EventDataQuery.ts

```diff
@@ -0,0 +1,38 @@
+import {gql} from "@apollo/client";
+
+export const GET_EVENT_QUERY = gql`
+    query getEvent($id: String!) {
+        event(id: $id) {
+            id
+            title
+            subtitle
+            author {
+                nickname
+            }
+            timeFrame {
+                startDate
+            }
+            theme {
+                primaryColor
+                image
+            }
+        }
+    }
+`;
+
+export interface EventData {
```

Comment:
Whe it's called EventData. IMO Event would be better but there may be some logic behind this name...

Reply:
It's supposed to hold types for Query Result, but due to my hate to using multiples interfaces/classes for one data type i'm considering to make it universal but as DOP instead of OOP